components/style/traversal.rs

```diff
@@ -201,6 +211,7 @@ pub fn recalc_style_at<'a, N, C>(context: &'a C,
     let mut bf = take_thread_local_bloom_filter(parent_opt, root, context.shared_context());
 
     let nonincremental_layout = opts::get().nonincremental_layout;
+    let mut restyle_result = RestyleResult::Continue;
     if nonincremental_layout || node.is_dirty() {
```

Comment:
Hmm... I can't see what you mean, you still need to insert the node in the bloom filter, complete animations, etc.

The rest of the comments are addressed or were on outdated commits and no longer apply I think.